src/Raven.Pal/inc/rvn.h

```diff
@@ -37,6 +37,13 @@ enum
     OPEN_FILE_DO_NOT_MAP = (1<<9)
 };
 
+enum
```

Comment:
@ayende NuGet is aching packages, you need to update the version of the Raven.PAL, it now goes from cache on CI

Reply:
Not a problem for me, just means that on release, we'll not have 6.2.0 but something like 6.2.138